test/integration/functional_test.go

```diff
@@ -564,6 +566,63 @@ func validateMySQL(ctx context.Context, t *testing.T, profile string) {
 	}
 }
 
+// validateFileSync start up VM and test sync file between host and VM
+func validateFileSync(ctx context.Context, t *testing.T, profile string) {
+	// 1. create dummy folder for MINIKUBE_HOME
+	oldMinikubeHome := os.Getenv("MINIKUBE_HOME")
+	defer os.Setenv("MINIKUBE_HOME", oldMinikubeHome)
+
+	home := os.Getenv("HOME")
+	fmt.Println("HOME is ", home)
+	minikubeHome, err := ioutil.TempDir(home, "")
+	if err != nil {
+		t.Fatalf("error during creating tmp dir: %v", err)
+	}
+	defer os.RemoveAll(minikubeHome)
+	os.Setenv("MINIKUBE_HOME", minikubeHome)
+	fmt.Println("Setting  MINIKUBE_HOME  ", minikubeHome)
+
+	// 2. start VM
+	startArgs := append([]string{"start", "-p", profile, "--wait=true", "--alsologtostderr", "-v=3"}, StartArgs()...)
+	c := exec.CommandContext(ctx, Target(), startArgs...)
+	rr, err := Run(t, c)
+	if err != nil {
+		t.Errorf("%s failed: %v", rr.Args, err)
+	}
+
+	// 3. copy random file to MINIKUBE_HOME/files/etc
+	tempfile := filepath.Join(minikubeHome, ".minikube/files/etc/filesync.txt")
```

Comment:
can you double check if ".minikube" is needed ? and also dont hard code directories. you could use filepath.Join

Reply:
for example from other integration tests I see this ``` fp := filepath.Join(localpath.MiniPath(), "cache", "images", img) ``` ...